styles/info.module.css

```
@@ -0,0 +1,4 @@
+.container {
+  display: grid;
+  grid-template-rows: 40px auto;
```

```suggestion
  grid-template-rows: 2.6rem auto;
```
Maybe you can use rem here instead of px? Just a suggestion but u know it better 👍